assets/javascripts/kitten/components/cards/contribution-card/components/input.js

```diff
@@ -1,39 +1,26 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
 import { TextInputWithUnit } from '../../../form/text-input-with-unit'
-import { domElementHelper } from '../../../../helpers/dom/element-helper'
 
-export const Input = ({value, ...props}) => {
+export const Input = props => {
   const [isInputEmpty, setEmptyInput] = useState(true)
 
-  useEffect(() => {
-    setTimeout(() => {
-      if (!domElementHelper.canUseDom()) return
-  
-      const inputElement = document.getElementById('inputElement')
-  
-      inputElement.focus()
-      if (inputElement.value) {
-        const valueLength = inputElement.value.length
-        inputElement.setSelectionRange(valueLength, valueLength + 1)
-      }
-    }, 500)
-  }, [])
-
   return (
     <TextInputWithUnit
-      id="inputElement"
-      defaultValue={value}
+      wrapperProps={{
+        className: classNames(
+          { 'k-ContributionCard__input--isEmpty': isInputEmpty },
```

Comment:
je changerais le nom de la classe CSS étant donné qu'elle s'applique au wrapper maintenant

```suggestion
          { 'k-ContributionCard__inputWrapper--isEmpty': isInputEmpty },
```